docs/source/en/add_new_model.md

```diff
@@ -541,6 +541,46 @@ input_ids = tokenizer(input_str).input_ids
 
 When both implementations have the same `input_ids`, add a tokenizer test file. This file is analogous to the modeling test files. The tokenizer test files should contain a couple of hardcoded integration tests.
 
+## Implement image processor
+
+> [!TIP]
+> We recommend adding a fast image processor (inheriting from [`BaseImageProcessorFast`]) in addition to the 'slow' image processor ([`BaseImageProcessor`]) to provide users with the best performance. Feel free to tag [@yonigozlan](https://github.com/yonigozlan) for guidance on adding a [`BaseImageProcessorFast`].
+> Fast image processors uses the [torchvision](https://pytorch.org/vision/stable/index.html) library and can perform image processing on the GPU, significantly improving processing speed.
+
+While this example doesn't include an image processor, you may need to implement one if your model requires image inputs. The image processor is responsible for converting images into a format suitable for your model. Before implementing a new one, check whether an existing image processor in the Transformers library can be reused, as many models share similar image processing techniques. Note that you can also use [modular](./modular_transformers) for image processors to reuse existing components.
```

Comment:
```suggestion
If your model requires image inputs, you need to implement an image processor. The image processor converts images into the models expected input format. Before implementing a new one, check whether an existing image processor in the Transformers library can be reused because many vision models share similar image processing techniques.

You can also implement an image processor the [modular](./modular_transformers) way because they reuse existing components.
```